tutor/src/screens/grading-templates/editors.js

```diff
@@ -300,6 +313,79 @@ class TemplateForm extends React.Component {
         );
     }
 
+    renderMultipleAttemptsFields(form, template) {
+        const header = (
+            <>
+                <Line />
+                <Row>
+                    Allow multiple-attempts
+                    <HintText>
+                        To enable, feedback must be set to <i>‘immediately after student answers’</i>.
+                    </HintText>
+                </Row>
+            </>
+        );
+
+        const iconText = (
+            <>
+                <strong>Example:</strong>
+                <div>MCQ with 4 choices, students get 2 (4-2) attempts.</div>
+                <div>MCQ with 3 choices, students get 1 (3-2) attempt.</div>
+                <div>MCQ with 2 choices, students get 1 attempt only.</div>
+            </>
+        );
+
+        const body = (
+            <>
+                <FieldsetRow
+                    legend="For auto-graded questions"
+                    legendHint={
+                        <>
+                            Attempts allowed is equal to no. of choices ‘minus’ 2
+                            <InfoIcon tooltip={iconText} />
+                        </>
+                    }
+                >
+                    <div>
+                        <Setting>
+                            <RadioInput
+                                name="allow_auto_graded_multiple_attempts"
+                                label="Yes"
+                                value={true}
+                                defaultChecked={template.allow_auto_graded_multiple_attempts == true}
+                            />
+                        </Setting>
+                        <Setting>
+                            <RadioInput
+                                name="allow_auto_graded_multiple_attempts"
+                                label="No"
+                                value={false}
+                                defaultChecked={template.allow_auto_graded_multiple_attempts == false}
+                            />
+                        </Setting>
+                    </div>
+                </FieldsetRow>
+                <Row>
+                    {form.values.allow_auto_graded_multiple_attempts == 'true' &&
```

Comment:
Is it weird for this to be the string "true"?

Reply:
Yeah it's Formik looking at the HTML attribute for the value, which is fine, but I'm glad you pointed it out because it made me discover a bug with the message not appearing when editing an existing template!

Also getting a boolean makes it read better: https://github.com/openstax/tutor-js/pull/3713/commits/2c38432c60f86a6263353347cb8cbb613bc5eef5